src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -623,6 +623,11 @@ internal bool LoadNodeLibrary(string library, bool isExplicitlyImportedLib)
             }
             catch (Exception e)
             {
+                // If the exception is having HRESULT of 0x80131515, then we need to instruct the user to "unblock" the downloaded DLL.
+                if (e is FileLoadException fle && fle.HResult == unchecked((int)0x80131515))
```

Comment:
do we have a common, low level utility where we can put this exception check ?
something like ExceptionUtils.IsBlockedDllException(Exception e) ?

Reply:
Can do